src/network.zig

```diff
@@ -2345,3 +2382,41 @@ const ProtocolTask = struct {
 		main.globalAllocator.destroy(self);
 	}
 };
+
+test "Parse address" {
+	const address = try IpAddress.fromString("127.0.0.1", .parse, undefined);
+
+	try std.testing.expectEqual(IpAddress.localHost, address);
+	const socketAddress = try SocketAddress.fromString("127.0.0.1:1234", .parse, undefined);
+	try std.testing.expectEqual(SocketAddress{.ip = IpAddress.localHost, .port = 1234}, socketAddress);
```

Comment:
Please compare with an explicit constant, otherwise this test becomes tautological with the suggested changes to the localhost constant.